sql/pre_install/tables.sql

```diff
@@ -195,8 +195,14 @@ CREATE TABLE IF NOT EXISTS _timescaledb_config.bgw_job (
     job_type            NAME        NOT NULL,
     schedule_interval   INTERVAL    NOT NULL,
     max_runtime         INTERVAL    NOT NULL,
-    max_retries         INT         NOT NULL,
+    max_retries         INTEGER     NOT NULL,
     retry_period        INTERVAL    NOT NULL,
+    proc_name           NAME        NOT NULL DEFAULT '',
+    proc_schema         NAME        NOT NULL DEFAULT '',
+    owner               NAME        NOT NULL DEFAULT CURRENT_ROLE,
```

Comment:
Are we adding DDL around job ownership/privileges? Could you elaborate why we need owner information?

Reply:
thats the user custom jobs will run as

Reply:
current policies all have an owner which is implicit from the job type, so for current policies this would just make it explicit

Reply:
the job gets executed as owner as background job, alter job can be done by any member of the role, for job function execution outside of background worker normal postgres permissions apply

Reply:
But the functions would still execute with the permissions of the definer when marked SECURITY DEFINER right? Even when executed by the background job?

Reply:
yes